infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -506,7 +528,17 @@ public void prepareSiteRoleChange() {
      */
     @Override
     public int queryReplicationStatus() {
-        return replicationManager.queryReplicationStatus();
+        if (localClusterDescriptor.getRole() == ClusterRole.ACTIVE) {
```

Comment:
Does this calculate how much is pending or how much is done? Can we fix the method description and rename accordingly.

Reply:
I think it calculates how much is pending and have put some comments... I think it would be better to get addressed by another PR.

Reply:
Can you add a TODO in the code please so we don't miss this whenever we decide to merge into master.